docs/help-center/tim-features/login.mdx

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_label: Login
+title: Login
+hide_title: true
+displayed_sidebar: helpcenterSidebar
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Login to a TIM server</h2>
+  </div>
+</div>
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop">
+  <ol>
+    <li>Open Famedly web app.</li>
+    <li>Enter your login URL into the corresponding input field.</li>
+    <li>Click <b>Next</b>.</li>
+    <li>A new tab for entering the login credentials will open.</li>
+    <li>Enter your username and click **Continue**.</li>
+    <li>Enter your password and click **Continue**.</li>
+    <li>You might get asked for a second factor, copy the second factor from your mails or authenticator app into the input field for the code and click **continue**.</li>
```

Comment:
The second factor is mandatory for TIM. I think we can just say 'enter your second factor', instead of might.